pyocd/tools/pyocd.py

```diff
@@ -896,10 +896,10 @@ def do_write(self, args, width):
             # Look up flash region.
             region = self.session.target.memory_map.get_region_for_address(addr)
             if not region:
-                print("address 0x%08x is not within a memory region", pagaddre_addr)
+                print("address 0x%08x is not within a memory region" % pagaddre_addr)
```

Comment:
Since you're already touching on this line, would you mind fixing the bad `pagaddre_addr` variable name (I just noticed it)?

Reply:
Done!

Reply:
Thanks!